File: mockmate-backend/src/main/resources/runners/python/runner_int+string.py
import sys
import json

def main():
    input_data = sys.stdin.read().strip()
    if not input_data:
        return
    parts = input_data.split(maxsplit=1)
    if len(parts) == 0:
        return

    k = int(parts[0])
    s = parts[1].strip() if len(parts) > 1 else ""
    if s.startswith('"') and s.endswith('"'): s = s[1:-1]
    elif s.startswith("'") and s.endswith("'"): s = s[1:-1]

    sol = Solution()
    result = sol.{{methodSignature}}(k, s)

    if isinstance(result, bool):
        print(str(result).lower())
    else:
        print(result)

{{USER_CODE}}

if __name__ == '__main__':
    main()

File: mockmate-backend/src/main/resources/runners/java/runner_int+string.java
import java.util.*;
import java.util.regex.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try {
            int k = sc.hasNextInt() ? sc.nextInt() : 0;
            String s = "";
            if (sc.hasNext()) {
                s = sc.nextLine().trim();
                // strip leading commas/spaces
                s = s.replaceFirst("^[,\\s]+", "");
                if(s.startsWith("\"") && s.endsWith("\"")) s = s.substring(1, s.length()-1);
            }
            Solution sol = new Solution();
            Object result = sol.{{methodSignature}}(k, s);
            System.out.println(result);
        } catch (Exception e) {
            e.printStackTrace();
            System.exit(1);
        } finally {
            sc.close();
        }
    }
}

{{USER_CODE}}
